plugins/modules/win_psrepository_copy.py

```diff
@@ -55,6 +55,11 @@
       - systemprofile
       - LocalService
       - NetworkService
+  force:
+    description:
+      - If C(True), this deletes repositories that are not present in the source.
+    type: bool
+    default: False
```

Comment:
We need to include the version when this new option was first present in which will be 2.4.0.

```suggestion
    default: False
    version_added: 2.4.0
```